packages/next/export/worker.js

```diff
@@ -213,7 +213,7 @@ export default async function ({
     await writeFileP(htmlFilepath, html, 'utf8')
     return results
   } catch (error) {
-    console.error(`\nError occurred prerendering ${path}:`, error)
+    console.error(`\nError occurred prerendering page "${path}":`, error)
```

Comment:
```suggestion
    console.error(`\nError occurred prerendering path "${path}":`, error)
```
Won't calling this `page` add some confusion since we're logging the `path` not the `page`?